ui/lib/theme.ts

```diff
@@ -2,6 +2,9 @@
 import { createTheme } from "@material-ui/core";
 // eslint-disable-next-line
 import { createGlobalStyle, DefaultTheme } from "styled-components";
+//@ts-ignore
+import bg from "../images/General-backgroundok.png";
```

Comment:
How about we move this to `lib/images`?

Reply:
Move the images folder inside lib?

Reply:
Wow nevermind, I had my branches crossed. Carry on!